internal/serverless/project.go

```diff
@@ -232,36 +232,49 @@ func (p *Project) getFleetHealth(ctx context.Context) error {
 	return nil
 }
 
-func (p *Project) CreateAgentPolicy(stackVersion string, kibanaClient *kibana.Client, outputId string) error {
-	systemPackages, err := registry.Production.Revisions("system", registry.SearchOptions{
-		KibanaVersion: strings.TrimSuffix(stackVersion, kibana.SNAPSHOT_SUFFIX),
-	})
-	if err != nil {
-		return fmt.Errorf("could not get the system package version for Kibana %v: %w", stackVersion, err)
-	}
-	if len(systemPackages) != 1 {
-		return fmt.Errorf("unexpected number of system package versions for Kibana %s - found %d expected 1", stackVersion, len(systemPackages))
-	}
-	logger.Debugf("Found %s package - version %s", systemPackages[0].Name, systemPackages[0].Version)
-
+func (p *Project) CreateAgentPolicy(kibanaClient *kibana.Client, stackVersion string, outputId string, selfMonitor bool) error {
 	policy := kibana.Policy{
 		ID:                "elastic-agent-managed-ep",
 		Name:              "Elastic-Agent (elastic-package)",
 		Description:       "Policy created by elastic-package",
 		Namespace:         "default",
-		MonitoringEnabled: []string{"logs", "metrics"},
+		MonitoringEnabled: []string{},
```

Comment:
Well, maybe we need logs collection to catch errors in logs :thinking:

Reply:
Checking about the system tests and the Agent policy created there does not have logs and metrics enabled.
https://github.com/elastic/elastic-package/blob/4af13a5c73432b0d7b20f595adcb1acaa8228eff/internal/testrunner/runners/system/runner.go#L560-L568

So even if the `Elastic-Agent (elastic-package)` policy has those settings enabled, when the system agent policy is assigned to that agent those logs/metrics are not collected

![agent with system test policy](https://github.com/elastic/elastic-package/assets/5330827/3dfeb01f-bd29-49e5-8458-dd76ecdfb03e)